web-common/src/features/project/ProjectDeployer.ts

```diff
@@ -165,7 +172,13 @@ export class ProjectDeployer {
       projectResp.localProjectName,
       checkNextOrg,
     );
-    if (frontendUrl) window.open(frontendUrl + "/-/invite", "_self");
+    const projectUrl = frontendUrl; // https://ui.rilldata.com/<org>/<project>
```

Comment:
Why not just rename the above to `projectUrl`?

Reply:
Agreed, just changed